manage.py

```diff
@@ -2,27 +2,9 @@
 import os
 import sys
 
-# Now we can import from third-party libraries.
 from django.core.management import execute_from_command_line
 
-# Don't try to force a new setting module on us
-if 'DJANGO_SETTINGS_MODULE' not in os.environ:
-
-    # use settings_test.py for running tests
-    if 'test' in sys.argv:
-        settings_mod = 'settings_test'
-    else:
-        # or try the optional "local" settings module first
-        try:
-            import settings_local  # noqa
-        except ImportError:
-            # or use the plain settings.py module
-            settings_mod = 'settings'
-        else:
-            settings_mod = 'settings_local'
-
-    # override the env var with what we want
-    os.environ['DJANGO_SETTINGS_MODULE'] = settings_mod
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'kuma.settings.local')
```

Comment:
This requires changes to the testing chapter in the docs. It also probably requires some thinking. Doing:

```
make django-tests
```

Doesn't allow you to pass along args (and doesn't have the flags) so it's much harder to use for development. Maybe we should instead create a `test.sh` bash script something like this?:

```
#!/bin/bash

export DJANGO_SETTINGS_MODULE=settings_test
./manage.py test -s --noinput --logging-clear-handlers $@
```